DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -93,10 +93,11 @@ public function process(ContainerBuilder $container)
                 }
 
                 $groupDefaults[$resolvedGroupName]['items'][] = array(
-                    'admin'        => $id,
-                    'label'        => !empty($attributes['label']) ? $attributes['label'] : '',
-                    'route'        => '',
-                    'route_params' => array(),
+                    'admin'          => $id,
+                    'label'          => !empty($attributes['label']) ? $attributes['label'] : '',
+                    'route'          => '',
+                    'route_params'   => array(),
+                    'route_absolute' => true,
```

Comment:
@core23 @andrey1s why was this merged? `route_absolute` still defaults to `true` cc @fracsi , see #4175